bigtest/tests/provider-edit-token-test.js

```diff
@@ -0,0 +1,189 @@
+import { expect } from 'chai';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+
+import { describeApplication } from '../helpers/describe-application';
+import ProviderShowPage from '../interactors/provider-show';
+import ProviderEditPage from '../interactors/provider-edit';
+
+describeApplication('ProviderEditToken', () => {
+  let provider,
+    packages,
+    longToken;
+
+  beforeEach(function () {
+    provider = this.server.create('provider', 'withPackagesAndTitles', 'withTokenAndValue', {
+      name: 'League of Ordinary Men',
+      packagesTotal: 5,
+      packagesSelected: 3
+    });
+
+    packages = this.server.schema.where('package', { providerId: provider.id }).models;
+    packages[0].visibilityData.isHidden = true;
+  });
+
+  describe('visiting the provider edit page with a token and value ', () => {
+    beforeEach(function () {
+      return this.visit(`/eholdings/providers/${provider.id}/edit`, () => {
+        expect(ProviderEditPage.$root).to.exist;
+      });
+    });
+
+    it('has token help text', () => {
+      expect(ProviderEditPage.tokenHelpText).to.equal('Enter your Gale token');
+    });
+
+    it('has token help prompt', () => {
```

Comment:
another minor nitpick here: probably the name is supposed to be `has token prompt` instead of `has token help prompt`?

Reply:
Yes -- corrected to has token prompt